test/validator/astnode-cp/ASTNodeOperation.test.ts

```diff
@@ -947,27 +947,27 @@ describe('ASTNodeOperation', () => {
 			});
 			it('compound types.', () => {
 				const goal: AST.ASTNodeGoal = AST.ASTNodeGoal.fromSource(`
-					let a: unknown = [];
-					let b: unknown = [42];
-					let c: unknown = [x= 42];
-					let d: Object  = List.<int>([]);
-					let e: Object  = List.<int>([42]);
-					let f: Object  = Dict.<int>([x= 42]);
+					let a: unknown = ();
+					let b: unknown = (42,);
+					let c: unknown = (x= 42);
+					let d: Object  = List.<int>(());
+					let e: Object  = List.<int>((42,));
+					let f: Object  = Dict.<int>((x= 42));
```

Comment:
```suggestion
					let d: Object  = [];
					let e: Object  = [42];
					let f: Object  = [x= 42];
```
same for testing operators below